docs/topics/mta-web-creating-migration-waves.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * docs/web-console-guide/master.adoc
+
+:_content-type: PROCEDURE
+[id="mta-web-creating-migration-waves_{context}"]
+= Creating migration waves
+
+A migration wave is a tool for creating Jira issues for a group of applications that are to be migrated at the same time. For each application assigned to a migration wave, {ProductShortName} opens a separate Jira issue with an automatically generated title.
```

Comment:
The primary goal of a migration wave is to offer the user a way to group applications to be migrated on a given schedule. Each application of a migration wave can be exported to external issue management system (only Jira for now).

Reply:
Corrected